src/components/tile/tile.test.js

```diff
@@ -270,11 +222,171 @@ context("Tests for Tile component", () => {
+        </Tile>
+      );
+
+      cy.checkAccessibility();
+    });
+
+    it.each(testData)(
+      "should check Accessibility for ddTextAlign as %s for Tile component",
+      (align) => {
+        CypressMountWithProviders(
+          <testStories.DlTileComponent ddTextAlign={align} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each(testData)(
+      "should check Accessibility for dtTextAlign as %s for Tile component",
+      (align) => {
+        CypressMountWithProviders(
+          <testStories.DlTileComponent
+            dtTextAlign={align}
+            ddTextAlign="right"
+          />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it("should check Accessibility for single column for Tile component", () => {
+      CypressMountWithProviders(
+        <testStories.DlTileComponent dtTextAlign="left" asSingleColumn />
+      );
+
+      cy.checkAccessibility();
+    });
+
+    it.each([10, 30])(
+      "should check Accessibility for dtTextAlign as %s for Tile component",
+      (w) => {
+        CypressMountWithProviders(
+          <testStories.DlTileComponent
+            w={w}
+            dtTextAlign="left"
+            ddTextAlign="left"
+          />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each(["default", "transparent", "black"])(
+      "should check Accessibility for tile footer variant as %s for Tile component",
+      (variant) => {
+        CypressMountWithProviders(
+          <testStories.TileFooterComponent variant={variant} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS])(
+      "should check Accessibility for tile footer children as %s for Tile component",
+      (children) => {
+        CypressMountWithProviders(
+          <Tile>
+            <TileFooter p={3}>
+              <Typography pr={2} display="inline" variant="b">
+                {children}
+              </Typography>
+            </TileFooter>
+          </Tile>
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([10, 30, 60])(
+      "should check Accessibility for w as %s for Tile component",
+      (w) => {
+        CypressMountWithProviders(<testStories.DlTileComponent w={w} />);
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each(["default", "selected", "positive", "negative", "caution", "info"])(
+      "should check Accessibility for border variant as %s for tile component",
+      (borderVariant) => {
+        CypressMountWithProviders(
+          <testStories.TileComponent borderVariant={borderVariant} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([
+      "borderWidth000",
+      "borderWidth100",
+      "borderWidth200",
+      "borderWidth300",
+      "borderWidth400",
+    ])(
+      "should check Accessibility for border width as %s for tile component",
```

Comment:
"should check Accessibility for border width as %s for **Tile** component",

Reply:
updated